lib/mayaUsd/undo/UsdUndoStateDelegate.h

```diff
@@ -79,20 +95,28 @@ class MAYAUSD_CORE_PUBLIC UsdUndoStateDelegate : public SdfLayerStateDelegateBas
         const SdfAbstractDataConstValue& value) override;
 
     void _OnSetTimeSample(const SdfPath& path, double time, const VtValue& value) override;
-    void _OnSetTimeSample(const SdfPath& path, double time, const SdfAbstractDataConstValue& value) override;
+    void _OnSetTimeSample(const SdfPath& path, double time, const SdfAbstractDataConstValue& value)
+        override;
 
     void _OnCreateSpec(const SdfPath& path, SdfSpecType specType, bool inert) override;
 
     void _OnDeleteSpec(const SdfPath& path, bool inert) override;
     void _OnMoveSpec(const SdfPath& oldPath, const SdfPath& newPath) override;
 
-    void _OnPushChild(const SdfPath& parentPath, const TfToken& fieldName, const TfToken& value) override;
-    void _OnPushChild(const SdfPath& parentPath, const TfToken& fieldName, const SdfPath& value) override;
-    void _OnPopChild(const SdfPath& parentPath, const TfToken& fieldName, const TfToken& oldValue) override;
-    void _OnPopChild(const SdfPath& parentPath, const TfToken& fieldName, const SdfPath& oldValue) override;
+    void _OnPushChild(const SdfPath& parentPath, const TfToken& fieldName, const TfToken& value)
+        override;
+    void _OnPushChild(const SdfPath& parentPath, const TfToken& fieldName, const SdfPath& value)
+        override;
+    void _OnPopChild(const SdfPath& parentPath, const TfToken& fieldName, const TfToken& oldValue)
+        override;
+    void _OnPopChild(const SdfPath& parentPath, const TfToken& fieldName, const SdfPath& oldValue)
+        override;
 
 private:
-    void _OnSetFieldDictValueByKeyImpl(const SdfPath& path, const TfToken& fieldName, const TfToken& keyPath);
+    void _OnSetFieldDictValueByKeyImpl(
+        const SdfPath& path,
+        const TfToken& fieldName,
+        const TfToken& keyPath);
     void _OnSetTimeSampleImpl(const SdfPath& path, double time);
 
```

Comment:
Super ugly after the clang-format is applied!!!!!! Something to fix at later time.